README.md

```diff
@@ -1,17 +1,16 @@
-# aa-toolbox: Toolbox for anticipatory action
+# OCHA AnticiPy: Get the data you need for anticipating humanitarian risk
```

Comment:
Indeed I don't love it either. I just wanted to avoid saying "anticipatory action" because I think the point is that we want to be a bit broader. 

I like your suggestion "Access data", I will make that change. The rest I wil leave for now, I think there is an idea next year to polish up some of the text together with comms before we do any kind of promotion, so there will still be a chance to change the text we don't like.